### PR TITLE
Guidelines for all organization repositories

### PR DESCRIPTION
**Pull request by robertodr**
## Description

I have added three markdown files that apply to **all** repositories in the organization:
* a code of conduct, based on the [Contributor Covenant](https://www.contributor-covenant.org)
* a very short guide to get support
* a slightly longer set of contribution guidelines.

## Todos

* **Developer Interest**

  - [X] Added `CODE_OF_CONDUCT.md`
  - [X] Added `SUPPORT.md`
  - [X] Added `CONTRIBUTING.md`

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Discussion points before merging

- [x] We need to have an email address in the `CODE_OF_CONDUCT.md` where people can file violation complaints.
~- [ ] We should discuss which license should be applied to our code and if contributions should be accompanied by signing of a [contributor license agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement) or a [developer certificate of origin](https://developercertificate.org/) More reading material:~
   ~* https://opensource.com/article/18/3/cla-vs-dco-whats-difference~
   ~* https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/~
~- [ ] The contribution guidelines are skewed towards contributions from people *outside* the organization. Thus they talk about forking repositories which, internally, we do not do. We can rephrase to cover both cases or we might move that section altogether to a page on the (yet-to-be) knowledge base. These figures might anyway come in handy: https://psicode.org/psi4manual/master/build_obtaining.html#faq-forkpsi4public~

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Contributing to Algorithmiq
+
+The following are guidelines and suggestions for contributing to Algorithmiq's repositories.
+
+## Getting started
+
+Please see our [website](https://github.com/Algorithmiq) for a brief overview of our projects.
```
Comment (matteoacrossi) [inline CONTRIBUTING.md:7]:
```suggestion
Please see our [GitHub organization page](https://github.com/Algorithmiq) for a brief overview of our projects.
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (matteoacrossi) [inline CODE_OF_CONDUCT.md:63]:
```suggestion
[info@algorithmiq.fi]
```

**Review by matteoacrossi — APPROVED** [review]
I like this. For the `CONTRIBUTING` file, perhaps it makes things easier to discuss also the workflow that does not involve forking.

**Comment by robertodr** [comment]
We don't have a license yet, so it's left blank in the contributing guidelines. Left a note in #4 to follow up

**Pull request merged**